nf_core/modules/lint/module_tests.py

```diff
@@ -19,69 +19,85 @@ def module_tests(_, module):
     file.
 
     """
-
+    nftest_testdir = os.path.join(module.component_dir, "tests")
+    if os.path.exists(nftest_testdir):
+        module.passed.append(("test_dir_exists", "nf-test test directory exists", nftest_testdir))
     if os.path.exists(module.test_dir):
         module.passed.append(("test_dir_exists", "Test directory exists", module.test_dir))
     else:
         module.failed.append(("test_dir_exists", "Test directory is missing", module.test_dir))
+        module.failed.append(("test_dir_exists", "nf-test directory is missing", nftest_testdir))
```

Comment:
Does this throw a failure or a warning? I'm fine if we want to really push the hard shift, but I think just a warning for now might be the right direction.

Reply:
It will throw two failures right now, but yeah I agree maybe the nf-test failure should be a warning. I'll try to figure out how to do that :p

Reply:
Allright fixed!